tests/Bot.Tests/Clockify/Reports/ReportSummaryServiceTest.cs

```diff
@@ -108,7 +108,7 @@ public async void Summary_UserHasOneWorkspace_DoesNotMentionWorkspaces()
             string summary = await reportSummaryService.Summary(Channels.Telegram, userProfile, dateRange);
 
             // Assert
-            summary.Should().BeEquivalentTo($"You worked **2.28d** in 01 January 2020 - 01 March 2020" +
+            summary.Should().BeEquivalentTo($"Between 01 January 2020 - 01 March 2020, you worked **2.28d** (18) hours" +
```

Comment:
You should write the assertion with the new message format, like 2.28 days (18 hours).

Reply:
Hi @khaelys , thank you for your reply. I will fix that